tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -566,18 +535,18 @@ def test_work_order_workerEncryptionKey_special_character(self):
         result_response = self.test_obj.getresult(
             self.test_obj.build_request_output['request_obj'])
 
-        # assert (
-        #         verify_test(
-        #             result_response, 0,
-        #             self.test_obj.build_request_output['pre_test_output'],
-        #             self.test_obj.build_request_output['action_obj'])
-        #         is TestStep.SUCCESS.value)
-
         assert (
-                check_negative_test_responses(
-                    submit_response,
-                    "Invalid Request")
+                verify_test(
+                    result_response, 0,
+                    self.test_obj.build_request_output['pre_test_output'],
+                    self.test_obj.build_request_output['action_obj'])
                 is TestStep.SUCCESS.value)
+
+        # assert (
```

Comment:
Remove the commented code if not needed

Reply:
changes addressed in commit "3010f9e43d2d6a3cea2f19a464bd6efbae48e964"